### PR TITLE
Add scss dropdown

### PR DESCRIPTION
**Pull request by ssb-cgn**

### CODE REVIEW

File: src/components/Dropdown/_dropdown.scss [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      }
+    }
+
+    input {
+      border: 1px solid $ssb-dark-6;
+      cursor: pointer;
+      font-size: 16px;
+      height: 36px;
+      padding: 4px 34px 4px 10px;
+      background-color: $ssb-white;
```
Comment (kristofferlund) [inline src/components/Dropdown/_dropdown.scss]: Sort alphabetically

File: src/components/Dropdown/_dropdown.scss [hunk 2]
```diff
@@ -0,0 +1,114 @@
+.ssb-dropdown {
+  @include roboto;
+  display: flex;
+  flex-direction: column;
+  transition: all .25s ease-in-out;
+
+  label {
+    margin-bottom: 5px;
+    font-size: 14px;
```
Comment (kristofferlund) [inline src/components/Dropdown/_dropdown.scss]: Sort alphabetically

File: src/components/Dropdown/_dropdown.scss [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+      background: $ssb-dark-3;
+      border-radius: 3px;
+    }
+
+    ::-webkit-scrollbar-thumb:hover {
+      background: $ssb-dark-4;
+    }
+
+  }
+}
```
Comment (kristofferlund) [inline src/components/Dropdown/_dropdown.scss]: Add a newline at the on of file

File: src/components/Dropdown/_dropdown.scss (hunk 1 above)
Comment (ssb-cgn) [reply src/components/Dropdown/_dropdown.scss]: fixed

File: src/components/Dropdown/_dropdown.scss (hunk 2 above)
Comment (ssb-cgn) [reply src/components/Dropdown/_dropdown.scss]: fixed

File: src/components/Dropdown/_dropdown.scss (hunk 3 above)
Comment (ssb-cgn) [reply src/components/Dropdown/_dropdown.scss]: fixed

**Pull request merged**